### DISCUSSION TITLE
What is the history behind the name of Yosys?

### DISCUSSION THREAD

**Discussion by Sysynth** [YosysHQ/yosys#3806] [General]
What is the meaning of "Yosys" and what is the origin of its name?

**Comment by nakengelhardt** [comment]
Yosys stands for "Yosys Open SYnthesis Suite".

**Reply by Sysynth** [reply]
I know, but does the Yosys inside "Yosys Open SYnthesis Suite" have a meaning at all?

**Reply by zachjs** [reply]
It's a recursive acronym, like GNU.

**Reply by nakengelhardt** [reply]
According to Claire, the letter Y was chosen because she was thinking of all the tools named [Yet Another ...](https://en.wikipedia.org/wiki/Yet_another)

**Reply by Sysynth** [reply]
Right, I forgot about the existence of recursive acronyms. Thank you guys!
